interface-spec.md

```diff
@@ -18,17 +18,19 @@
 
 **Properties:**
 
-- `String .value` is refined by each interface which extends Node
+- None 
 
 **Methods:**
 
-- `.equals(Node other)` returns true if and only if the argument is a) of the same type b) has the same contents (value and, if applicable, type or language)
+- `.sameTerm(Node other)` returns true if and only if the argument is a) of the same type b) has the same contents (value and, if applicable, type or language)
 
-### IRI extends Node
+### Symbol extends Node
+
+A node named with a URI.  The node is not a URI itsef.
 
 **Properties:**
 
-- `String .value` the IRI as a string (example: `http://example.org/resource`)
+- `String .uri` the IRI naming the node as a string (example: `http://example.org/resource`)
```

Comment:
Discussed in the 2015-11-27 meeting: we prefer to keep `.value` for uniformity with literals, but we will also need to add a `.type` or similar to distinguish between IRIs/literals/blank nodes/variables.

However, this choice means that checking whether something is an IRI and reading its value requires two steps. Yet it avoids the problem where an object would have both `.value` and `.uri`.